src/inmanta/module.py

```diff
@@ -1380,11 +1380,14 @@ def __init__(self, path: str, autostd: bool = True, main_file: str = "main.cf",
             if not os.path.exists(self._metadata.downloadpath):
                 os.mkdir(self._metadata.downloadpath)
 
+        self.virtualenv: env.PythonEnvironment
         if venv_path is None:
-            venv_path = os.path.abspath(os.path.join(path, ".env"))
+            self.virtualenv = env.process_env
         else:
             venv_path = os.path.abspath(venv_path)
-        self.virtualenv = env.VirtualEnv(venv_path)
+            self.virtualenv = env.VirtualEnv(venv_path)
```

Comment:
I expect this might cause issue if another module us loaded with the same venv_path because this will be a new instance every time, causing a lot of venv-re-activations in a row.

Should we allow venv_path to be an actual VirtualEnv ?

Reply:
Additionally, in the past, this constructor was such that it has no side effects: if is very lightweight to construct a project and not use it.

I would like the `self.virtualenv.use_virtual_env()` to be moved to the `load`  method.

to make this easier, it would be good to add `use_virtual_env` to the ActiveEnv that just returns?